packages/react/src/components/OverflowMenu/OverflowMenu.tsx

```diff
@@ -297,13 +437,13 @@ class OverflowMenu extends Component {
     evt.stopPropagation();
     if (!this._menuBody || !this._menuBody.contains(evt.target)) {
       this.setState({ open: !this.state.open });
-      onClick(evt);
+      onClick();
```

Comment:
Could consumers be relying on the `evt` to be passed as it was before?

Reply:
I am passing the evt now, and changed the type signature to accept a parameter optionally.